src/components/NSLookupTool/NSLookupTool.tsx

```diff
@@ -7,50 +7,54 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "NsLookup";
-const description_userguide =
-    "The nslookup command is a tool used to query Domain Name System (DNS) servers and retrieve information about a specific domain or IP address." +
+/**
+ * Title and Description of the tool
+ */
+const title = "NSLookup";
+const description_Userguide =
+    "The NSLookup command is a tool used to query Domain Name System (DNS) servers and retrieve information about a specific domain or IP address." +
     "This command is an essential tool for network administrators and system engineers as it can be used to troubleshoot DNS issues and gather information about DNS configurations." +
     "How to use NSLookUp.\n\n" +
     "Step 1: Enter an IP or Web URL.\n" +
-    "       E.g. 127.0.0.1\n\n" +
+    " E.g. 127.0.0.1\n\n" +
     "Step 2: View the Output block below to view the results of the Scan.";
 
-interface FormValues {
-    ipaddress: string;
+// Form Value Interface
+interface FormValuesType {
+    ipAddress: string; // Ip Address that needs to be looked up
+    tutorial: string; //Tutorial Text
 }
 
 export function NSLookup() {
```

Comment:
I do not think  you need export because you export it later. 
```suggestion
function NSLookup() {
```